src/NosCore.GameObject/Providers/InventoryService/InventoryService.cs

```diff
@@ -354,13 +354,14 @@ public InventoryItemInstance DeleteFromTypeAndSlot(NoscorePocketType type, short
             // check if the item is a palced MinilandObject
             if (sourcetype == NoscorePocketType.Miniland)
             {
-                if (sourcePocket.ItemInstance is MinilandObjectDto || destinationPocket.ItemInstance is MinilandObjectDto)
+                //todo test this (looks weird)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)